### PR TITLE
Feature overview

### PR DESCRIPTION
**Pull request by NilsHolthey**
With this PR I create an overview page:

Userstory: https://github.com/NilsHolthey/capstone-project/issues/25

### CODE REVIEW

File: src/components/Overview/OverviewGrid.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	const todos = useStore(state => state.todos);
+	const doingIts = useStore(state => state.doingIts);
+
+	const lengthOfTodos = todos.length;
+	const legthOfDoingIts = doingIts.length;
+	const lengthOfDidIts = didIts.length;
+	const sumOfArrays = lengthOfTodos + legthOfDoingIts + lengthOfDidIts;
+
+	const progressTodo = Math.round(
+		(lengthOfTodos / (lengthOfTodos + legthOfDoingIts + lengthOfDidIts)) * 100
```
Comment (anitaschiller) [inline src/components/Overview/OverviewGrid.js:20]: Why don't you use `sumOfArrays` (or `sumOfTasks` if you decide to change it)?

```suggestion
		(lengthOfTodos / sumOfArrays) * 100
```

File: src/components/Overview/PercentageOverview.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}) {
+	const [progress, setProgress] = useState(0);
+	useEffect(() => {
+		setProgress(percentage);
+	}, [percentage]);
+
+	const viewBox = `0 0 ${size} ${size}`;
+	const radius = (size - strokeWidth) / 2;
+	const circumference = radius * Math.PI * 2;
+	const dash = (progress * circumference) / 100;
```
Comment (anitaschiller) [inline src/components/Overview/PercentageOverview.js:30]: Maybe I missed sth but I don't think you need the `useState` and `useEffect`: 

```suggestion
	const viewBox = `0 0 ${size} ${size}`;
	const radius = (size - strokeWidth) / 2;
	const circumference = radius * Math.PI * 2;
	const dash = (percentage * circumference) / 100;
```

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Nils, pretty cool and quite complex (including your styled-components). Did not find much to comment on, but some smaller stuff 😉.

**Comment by NilsHolthey** [comment]
Thank you Anita, I made some changes...

**Review by anitaschiller — APPROVED** [review]
Looks good to me :)

**Pull request merged**